### DISCUSSION TITLE
Is there a way prevent the displaying of all the graph points at the sametime, ever, even when hovering ?

### DISCUSSION THREAD

**Discussion by Shasdo** [kalkih/mini-graph-card#904] [Q&A] [answered]
Hello

**Is there a way prevent the displaying of all the graph points at the same time, ever, even when hovering ?**

For me, It can be overwhelming and not aestheticaly pleasant when there is too many of them.

But I don't want the behavior of the line to be changed, and still want to be able to get a previous value to be displayed, like it currently do, by running the cursor along the graph line and showing one point under it at a time.

I have try all config permutation I could think of with "points" and "show_points" attributes without sucess.
Either they are completly disabled and you can't get a previous value at a specific point, either they are temporary hidden, but they still are all revelead when the mouse hover the graph.

Am i missing something ?

Thanks

**Comment by ildar170975** [comment, marked as answer]
Unclear what you need. 
Points may be shown always or while hovering, or may be not shown at all. 
A state is shown for a selected point; if no points are displayed - then surely it is not possible to show a state for some point. 
It is not possible to show only ONE point; all of them are displayed, a user can select some point to see a state.

**Comment by Shasdo** [comment]
I don't see the point (pun intended) to show them all when hovering.  I don't want this behavior because once you have too much points, the results is useless and quite dsigraceful.

<img width="287" alt="points" src="https://user-images.githubusercontent.com/4083901/212467271-d3a66ca3-6019-4aee-8a27-c75c3da7ee54.png">

But, I still want to be able to run the mouse on the graph line to see data value one by one, like it already do.
<img width="287" alt="point_unique" src="https://user-images.githubusercontent.com/4083901/212467407-977268ff-9d89-4760-8227-1e402528b0bd.png">
_Moving the mouse left or right over the line will show the next point and value etc._

But disabling points  through parameters disable also the wanted behavior above.
Your explanations, confirm my previous conclusions so that will not be possible.

Is a development for this behavior possible ? Should I post a request ?

Thx for your answer

**Comment by ildar170975** [comment]
A state is displayed for some point. 
If it is not selected - nothing to show. 
To select a point - you need to SEE it. 
If no points are displayed - nothing to select. 

You may create a FR indeed. 
But this FR should contain a consistent algorithm.
